packages/dropdowns/src/elements/Autocomplete/Autocomplete.tsx

```diff
@@ -33,15 +34,15 @@ interface IAutocompleteProps extends HTMLAttributes<HTMLDivElement> {
 }
 
 const Autocomplete = React.forwardRef<HTMLDivElement, IAutocompleteProps>(
-  ({ children, inputRef: controlledInputRef, start, ...props }, ref) => {
+  ({ children, inputRef: controlledInputRef = null, start, ...props }, ref) => {
```

Comment:
Could you clarify why the addition of `null` default value is needed here?

Reply:
The reason for the `null` default for the input ref is for [`react-merge-refs` and to avoid passing in an `undefined` ref](https://github.com/gregberge/react-merge-refs/blob/main/src/index.tsx#L10).

Reply:
@Francois-Esquire note that the loose `!=` equality check in `react-merge-refs` should work without needing to modify this code.

Reply:
Could an alternative be to do a conditional check against `null` prior to using `mergeRefs`? I think that would satisfy the TypeScript compiler without changing the function signature.

Not a blocker, though, as `controlledInputRef` doesn't seem to be used elsewhere for now.

Reply:
@jzempel good point! Technically would be fine if it was `undefined` which was my initial concern. @hzhu I did return the fn signature however I wasn't able to completely remove the `null` from the `mergeRefs` call.